### ISSUE TITLE
Fail to compile (typescript)

### ISSUE THREAD

**Issue by SpencerTuft** [davideicardi/live-plugin-manager#11]
There is an error small error in PluginManager.ts that prevents tsc from finishing the compile.

```ts
if (!this.isValidPluginName(registryInfo.name)) {
	throw new Error(`Invalid plugin name '${name}'`);
}
```

I believe the variable in the template string should be `registryInfo.name`

```ts
if (!this.isValidPluginName(registryInfo.name)) {
	throw new Error(`Invalid plugin name '${registryInfo.name}'`);
}
```

I didn't create a pull request since I'm not 100% sure what the intention was just glancing at the code.

**Comment by SpencerTuft** [comment]
Here's the error I get when I compile:

![Screen Shot 2019-05-08 at 10 38 26 AM](https://user-images.githubusercontent.com/20521760/57393823-28c17e00-7181-11e9-9a4a-f263082ec918.png)

**Comment by davideicardi** [comment]
You are right, I think this is an error. Strange that I missed it ... probably `name` is referencing some other variable...

Just fixed it in 0.13.2!

thanks!
